src/cheri-pte-ext.adoc

```diff
@@ -202,7 +204,7 @@ When V=1 <<vsstatusreg_pte,vsstatus>>.UCRG is in effect.
 
 <<mstatusreg_pte,mstatus>>.UCRG also exists. Reading or writing it is equivalent to reading or writing <<sstatusreg_pte,sstatus>>.UCRG.
 
-NOTE: As there is no M-mode translation available in RISC-V, there is no current software use for <<mstatusreg_pte,mstatus>>.UCRG or an M-mode equivalent bit in the future.
+NOTE: There is no current software use for <<mstatusreg_pte,mstatus>>.UCRG.
```

Comment:
I mean, there is. It's the same bit as sstatus.UCRG, which is used, so by definition so is mstatus.UCRG. I do not think there's anything interesting to say here; everything S-mode has shows up in mstatus, so exists, and there's no additional M-mode state because it's about virtual memory, which M-mode doesn't deal with unless requesting to access memory as if in S/U-mode via mstatus.MPRV. Which becomes even clearer if the extension in question is named Svfoo.

Reply:
This would be like saying there is no use for mstatus.SUM, which is in some sense true, M-mode isn't directly messing with that bit for its own use. But S-mode is, and it's the exact same bit, existing even in mstatus. And this is important if you want to have M-mode do world switching between multiple S-mode worlds, because although M-mode doesn't care what the bit is, it still needs to be context switching it.

Reply:
yeah true - I'll simplify the text

Reply:
removed.